src/Card/CardList.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Card } from 'baltimorecounty-react-components';
```

Comment:
What do the `{` `}` braces do in this context?  It looks to be specific to the `React.Fragment` (HTML?) below.

Reply:
will explain in person